changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 19th December 2022
```

Comment:
I'm not sure this is an accurate description of the changes, but I have raised some questions below about the changes, those should be resolved first before re-examining the Changelog description.